interfaces/cython/cantera/ck2cti.py

```diff
@@ -822,7 +822,7 @@ class Sri(object):
     """
     A kinetic model of a phenomenological rate coefficient k(T, P) using the
     "SRI" formulation of the blending function :math:`F` using either 3 or
-    5 parameters. See :ref:`sec-sri-falloff`.
+    5 parameters. See https://cantera.org/science/reactions.html#sec-sri-falloff.
```

Comment:
I'm not sure if this will be possible, since the content being referred to has moved to the main site, and I'm not sure how to get that information into Sphinx at build-time of the API docs. I agree it would be nice to have the arbitrary title references back though...